src/main/java/CommandType.java

```diff
@@ -0,0 +1,4 @@
+public enum CommandType {
+    EXITDUKE, PRINTALLTASKS, ADDTODO, ADDDEADLINE,
+    ADDEVENT, MARKASDONE, DELETETASK, FINDTASK;
```

Comment:
According to the Java coding standard guide, under the Naming section, it says that constants should be all uppercase using underscore to separate words. Should there be an underscore between variables with multiple words in it? For example, `EXITDUKE` could instead be `EXIT_DUKE`?

Reply:
Thanks for the tips!!